src/autotype/mac/AutoTypeMac.cpp

```diff
@@ -488,3 +507,25 @@ void AutoTypeExecutorMac::execKey(AutoTypeKey* action)
     m_platform->sendKey(action->key, false);
     usleep(25 * 1000);
 }
+
+void AutoTypeExecutorMac::execClearField(AutoTypeClearField* action = nullptr)
+{
+    Q_UNUSED(action);
+
+    m_platform->sendKey(Qt::Key_Control, true, Qt::ControlModifier);
+    m_platform->sendKey(Qt::Key_Up, true, Qt::ControlModifier);
+    m_platform->sendKey(Qt::Key_Up, false, Qt::ControlModifier);
+    m_platform->sendKey(Qt::Key_Control, false);
+    usleep(25 * 1000);
+    m_platform->sendKey(Qt::Key_Shift, true, Qt::ShiftModifier);
+    m_platform->sendKey(Qt::Key_Control, true, Qt::ShiftModifier | Qt::ControlModifier);
+    m_platform->sendKey(Qt::Key_Down, true, Qt::ShiftModifier | Qt::ControlModifier);
+    m_platform->sendKey(Qt::Key_Down, false, Qt::ShiftModifier | Qt::ControlModifier);
+    m_platform->sendKey(Qt::Key_Control, false, Qt::ShiftModifier);
+    m_platform->sendKey(Qt::Key_Shift, false);
```

Comment:
@phoerious 
1. <kbd>Command</kbd> <kbd>Up</kbd>   - Move the cursor to the beginning of the field
2. <kbd>Shift</kbd> <kbd>Command</kbd> <kbd>Down</kbd>   - Select the text between the cursor and the end of the field
3. <kbd>Delete</kbd>   - Delete the selection